build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -5,9 +5,29 @@ data:
     mkdir -force c:/var/log/antrea
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
-    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+
+        # From containerd version 1.7 onwards, the servcieaccount directory, the ca.cert and token files will automatically be created.
+        $serviceAccountPath = "C:\var\run\secrets\kubernetes.io\serviceaccount"
+        if (-Not $(Test-Path $serviceAccountPath)) {
```

Comment:
I understand this is better, but will the original code fail if the path exists?

Does the PR fix anything or just improve code? This is important to understand what's the compatibility of previous Antrea releases.

Reply:
Yes, it will fail in containerd 1.7. If the path exist, related files can not be modified.

Reply:
Then how does it fix it if the PR just skips creating the path when it exists. The related files still cannot be modified, right?

Reply:
Yes it can't be modified in this case, do you think we should update the document to ask user to cleanup related folder in containerd 1.7?

Reply:
So basically there are three cases: 
1. If the paths exist and files are unchanged, the agent skip coping files.
2. If the paths don't exist, the agent copies files to destination.
3. If the path exist and files have been modified, there is no way to modify the file and the pod is expected to crash. The user should cleanup related paths by themselves.

Reply:
According to this link (https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/), with containerd versions <=1.6, the ca file and token of the service account only exists inside containers, but antrea-agent needs files under path "c:\var\run\secrets\kubernetes.io\serviceaccount" to create kubeClient, so we should copy the files to the target path. However, since containerd 1.7, containerd would automatically place the files under path "c:\var\run\secrets\kubernetes.io\serviceaccount" ( it is supposed to be 3 links to the same file), so we don't perform "copy" actions.

So the change here is to be consistent of the previous versions.